test/check-machines-create

```diff
@@ -2134,6 +2137,13 @@ vnc_password= "{vnc_passwd}"
         b.click("#firmware-dialog-apply")
         b.wait_not_present(".pf-v5-c-modal-box__body")
         b.wait_in_text("#vm-VmNotInstalled-firmware", "UEFI")
+        self.assertIn("<os firmware='efi'>", m.execute("virsh dumpxml VmNotInstalled"))
+
+        # auto-enabled software TPM (doesn't work on RHEL 8 or arch)
+        if not m.image.startswith("rhel") and not m.image.startswith("centos") and m.image != 'arch':
```

Comment:
Should this run on rhel-9?

Reply:
Argh, copy pasta :spaghetti: It absolutely should run on RHEL 9, yes. Updated, thanks for spotting!